code/game/objects/structures/holosign.dm

```diff
@@ -212,6 +216,7 @@
 		return
 
 	var/mob/living/M = AM
-	M.electrocute_act(15,"Energy Barrier", flags = SHOCK_NOGLOVES)
+	M.Knockdown(10)
+	M.electrocute_act(15,"Energy Barrier", flags = SHOCK_NOGLOVES | SHOCK_NOSTUN)
```

Comment:
Can't believe you dismissed my review, but that's fair since it was out of scope and I was being a goober.

Did you want to replace ``M`` with ``L`` or a more meaningful variable name, like ``victim``.
``var/mob/M`` and ``var/mob/living/L`` should give you an idea of what the procedure is. We'd prefer meaningful names of that, however.